pkg/observers/host/host_windows.go

```diff
@@ -0,0 +1,44 @@
+// +build windows
+
+package host
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/shirou/gopsutil/process"
+	"golang.org/x/sys/windows"
+)
+
+func (p processName) getName(proc *process.Process) (string, error) {
```

Comment:
While not strictly necessary since the map in the struct is a pointer itself, maybe make these pointer receivers just for principle of least surprise for somebody down the road?

Reply:
agree 👍

Reply:
done